### PR TITLE
feat: Add `hideOutput` method.

### PR DESCRIPTION
**Pull request by xiCO2k**
This PR adds the `hideOutput` method.

This is an important method to have for testing purposes, since using this, no output will be render on CLI.

## Usage

```php
use function Termwind\hideOutput;

// With Pest
beforeEach(fn () => hideOutput());

// With PHPUnit
class ExampleTest extends TestCase
{
    public function setUp(): void
    {
        hideOutput();
    }
}
```

### CODE REVIEW

File: src/Functions.php [hunk 1]
```diff
@@ -20,6 +21,16 @@ function renderUsing(OutputInterface|null $renderer): void
     }
 }
 
+if (! function_exists('Termwind\hideOutput')) {
+    /**
+     * Hides the output for the CLI.
+     */
+    function hideOutput(): void
+    {
+        renderUsing(new BufferedOutput());
```
Comment (olivernybroe) [inline src/Functions.php:30]: You could consider using `NullOutput` instead actually 🤔 
https://github.com/symfony/console/blob/6.0/Output/NullOutput.php

Comment (xiCO2k) [reply src/Functions.php]: Even better! Thanks for that

File: tests/hideOutput.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+use function Termwind\hideOutput;
+use function Termwind\renderUsing;
+use Termwind\Termwind;
+
+beforeEach(fn () => renderUsing(null));
+
+it('can hide output', function () {
+    hideOutput();
+
+    expect(Termwind::getRenderer())->toBeInstanceOf(BufferedOutput::class);
```
[the remark this answers is not in the document]
Comment (xiCO2k) [reply tests/hideOutput.php]: not finding a good solution to this one, let me know if you have the solution to this one. Thanks @olivernybroe

[the remark this answers is not in the document]
Comment (olivernybroe) [reply tests/hideOutput.php]: If I find a clean way I'll just PR it later 👍

[the remark this answers is not in the document]
Comment (xiCO2k) [reply tests/hideOutput.php]: Awesome sounds great!

**Pull request closed without merging**